packages/core/src/models/executorOptions.ts

```diff
@@ -0,0 +1,3 @@
+export interface IExecutorOptions {
+  type?: string;
```

Comment:
Yes, it indicated the driver to use while sending queries, single driver design might be changed in the future. The property name here affects the user config, that is, "executor.type" v.s. "executor.executorType". So I think it would be better to use the name "type" and add some comments to the interface.

Reply:
Thanks for @oscar60310 !